Gems/PhysX/Code/Editor/CollisionLayerWidget.h

```diff
@@ -12,7 +12,7 @@
 #include <AzCore/Asset/AssetCommon.h>
 #include <AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI.h>
 #include <AzFramework/Physics/Collision/CollisionLayers.h>
-#include <Editor/ComboBoxEditButtonPair.h>
+#include <AzToolsFramework/UI/PropertyEditor/PropertyEnumComboBoxCtrl.hxx>
```

Comment:
Same here. Could this be a `AzToolsFramework::GenericComboBoxCtrl`?

Reply:
Hey thank you for your feedback :) I moved everything to the GenericComboBoxCtrl